lambda-alb/src/AWS/Lambda/ALB/Header.hs

```diff
@@ -1,40 +1,14 @@
 module AWS.Lambda.ALB.Header where
 
 import AWS.Lambda.Runtime.Prelude
-import Data.Aeson.KeyMap (KeyMap)
 
-import qualified Data.Aeson           as JSON
-import qualified Data.Aeson.Key       as JSON.Key
-import qualified Data.Aeson.KeyMap    as KeyMap
-import qualified Data.Aeson.Types     as JSON
 import qualified Data.CaseInsensitive as CI
 
+type Header     = (HeaderName, Text)
 type HeaderName = CI.CI Text
+type Headers    = [Header]
 
-type Header = (HeaderName, Text)
-
-newtype Headers = Headers [Header]
```

Comment:
@epicallan This newtype only ever was present to add json instances. But conceptually we only ever need that instances during parsing requests  and generating responses, and the format there is very specific to the request/responses. This mostly added indirection so defining that JSON instances directly on the request/responses is nicer IMO.

Reply:
> This mostly added indirection so defining that JSON instances directly on the request/responses is nicer IMO.

I agree with the removal.